node/node_test.go

```diff
@@ -269,7 +269,7 @@ func TestUpdateStakingWithdrawal(t *testing.T) {
 	if !ok {
 		t.Error("The correct address was not present")
 	}
-	if value != 10 {
+	if value != int64(1000) {
```

Comment:
why is it? constant?

Reply:
Please read the test carefully. It withdaws a stake of `10` from a current stake of `1010`, and thus checks whether `1000` was left.

Reply:
okay. this is test file - what i meant we need to write a better code because i saw you did not specify constant in other files.

Reply:
Added more comments to make this clear.